model/build.gradle

```diff
@@ -1,3 +1,36 @@
 plugins {
     id 'rush.java-library-conventions'
-}
\ No newline at end of file
+    id 'com.google.protobuf' version "0.8.18"
+    id 'java'
+}
+
+repositories {
+    gradlePluginPortal()
+}
+
+def grpcVersion = '1.44.1'
```

Comment:
we should keep all lib version in one place (it's gradle.settings for now) but we can do it later